itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -232,8 +232,6 @@ private Tuple replDumpDb(String dbName, String fromReplID, String toReplID, Stri
   }
```

Comment:
Do we have a test that checks the scenario we are fixing that is add partition followed by alter partition in incremental ?

Reply:
Yes, testEventTypesForDynamicAddPartitionByInsert will cover this case. Before this patch, there was a DROP TABLE added as a hack to avoid this bug. Now, since it is fixed, I modified this test case to test this scenario as well.